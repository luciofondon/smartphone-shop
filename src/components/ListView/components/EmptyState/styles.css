.empty-state{
    font-size: 40px;
    text-align: center;
    padding-top: 200px;
}