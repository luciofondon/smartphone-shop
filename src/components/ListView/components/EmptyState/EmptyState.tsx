import React from 'react';

import './styles.css';

const EmptyState = () => {
  return <div className="empty-state">No hay datos para mostrar</div>;
};

export default EmptyState;
