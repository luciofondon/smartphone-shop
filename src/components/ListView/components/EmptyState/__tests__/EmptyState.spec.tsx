import React from 'react';
import { render } from '@testing-library/react';
import EmptyState from '../EmptyState';

test('render EmptyState', () => {
  const { container } = render(<EmptyState />);
  expect(container).toMatchSnapshot();
});
