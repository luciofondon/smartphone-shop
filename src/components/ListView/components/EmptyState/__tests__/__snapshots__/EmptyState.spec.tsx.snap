// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render EmptyState 1`] = `
<div>
  <div
    class="empty-state"
  >
    No hay datos para mostrar
  </div>
</div>
`;
